setup.py

```diff
@@ -287,6 +288,21 @@ def setup_package():
                                   "installed.\n{}{}"
                                   .format(numpy_req_str, instructions))
 
+        scipy_status = get_package_status('scipy', SCIPY_MIN_VERSION)
+        scipy_req_str = "scikit-learn requires scipy >= {}.\n".format(
+            SCIPY_MIN_VERSION)
+
+        if scipy_status['up_to_date'] is False:
+            if scipy_status['version']:
+                raise ImportError("Your installation of "
+                                  "scipy {} is out-of-date.\n{}{}"
+                                  .format(numpy_status['version'],
+                                          numpy_req_str, instructions))
+            else:
+                raise ImportError("scipy is not "
+                                  "installed.\n{}{}"
+                                  .format(scipy_req_str, instructions))
+
```

Comment:
The code is highly redundant with the numpy one. I feel that it could be factored in the `get_package_status` function. But that could be done later, no need to delay the release.